### DISCUSSION TITLE
Unable to make whisper work with GPU

### DISCUSSION THREAD

**Discussion by dianis55** [openai/whisper#2194] [Q&A]
Hello! I'm trying to speed up whisper using GPU, but for some reason, it doesn't detect the video card.
Here's the code I used for testing:
```
import jax
import jax.numpy as jnp
print("JAX version:", jax.__version__)
print("JAX backend:", jax.lib.xla_bridge.get_backend().platform)
print("Available devices:", jax.devices())
x = jnp.ones((1000, 1000))
y = jnp.ones((1000, 1000))
result = jnp.dot(x, y)
print("Result of matrix multiplication:", result)

```

As a result, I get:
```
JAX backend: cpu
Available devices: [CpuDevice(id=0)]

```
However, the command nvcc --version returns the correct response for CUDA - 12.5
I believe I've set all the necessary environment variables.
What else could I be missing?
I'm working on Windows 11 with an NVIDIA 3080 Ti graphics card.

![image](https://github.com/openai/whisper/assets/107199631/d9d7359c-dead-4187-a132-add78609edc8)
![image](https://github.com/openai/whisper/assets/107199631/78207824-c8dd-4d5c-aae3-ece94e541a3d)

**Comment by phineas-pta** [comment]
u need `torch` not `jax` (also btw `jax` not support gpu on windows)
